.secrets.baseline

```diff
@@ -274,5 +274,5 @@
       }
     ]
   },
-  "generated_at": "2023-03-31T18:16:32Z"
+  "generated_at": "2023-04-04T19:16:10Z"
```

Comment:
This change seems unnecessary to me when no secrets have changed.

Reply:
Yep. I've run update-secrets and saw some changes in git status so just commited without checking that the date changed only.